.brackets.json

```diff
@@ -0,0 +1,6 @@
+{
+    "jscodehints.detectedExclusions": [
+        "C:/Users/Main/Documents/CodeForBoston/cambridge_voting_locations/app/bower_components/bootstrap-sass/assets/javascripts/bootstrap/popover.js",
```

Comment:
I'm guessing this file is from your code editor? Probably should not check this particular file into the repo.